org.eclipse.xtext.testing/tests/org/eclipse/xtext/testing/tests/junit5/RunJUnit5Suite.java

```diff
@@ -1,26 +0,0 @@
-/*******************************************************************************
```

Comment:
this will only work if you also pull/merge https://github.com/eclipse/xtext-core/compare/cd_junit5_call_4?expand=1

Reply:
@cdietrich remember that now your branch https://github.com/eclipse/xtext-core/compare/cd_junit5_call_4 has been superseded by this PR